tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/javainterop/basic/NegativeValidationTest.java

```diff
@@ -274,14 +274,14 @@ public void testMethodSignatureNotMatch7() {
     public void testMethodSignatureNotMatch8() {
         CompileResult compileResult = BCompileUtil.compile("test-src/javainterop/negative/distinct_error");
         compileResult.getDiagnostics();
-        Assert.assertEquals(compileResult.getDiagnostics().length, 3);
+        Assert.assertEquals(compileResult.getDiagnostics().length, 2);
         BAssertUtil.validateError(compileResult, 0,
```

Comment:
@warunalakshitha with this PR, we no longer show interop validation errors if there are errors in the package compilation. AFAIU, the purpose of this test is to validate interop and so IMO the package compilation should succeed so that the interop validation error can be asserted.

Is it what is expected with this test case or Is it expected to have compilation errors as well?

Reply:
Since it is a fail fast approach, showing interop errors once package compilation done seems ok fro me. BTW shall we modify test to fix the other compilation errors without changing the interop error assertion.

Reply:
+1 will change the test case to include the interop errors.